cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,70 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
-describe('Article', () => {
-  before(() => {
+import EditorPageObject from '../support/pages/editor.pageObject';
+import faker from 'faker';
+
+const editorPage = new EditorPageObject();
+let user;
+let article;
 
+describe('Article', () => {
+  beforeEach(() => {
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
   });
 
   beforeEach(() => {
     cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+      cy.login(user.email, user.username, user.password);
+    });
   });
 
   it('should be created using New Article form', () => {
-
+    editorPage.visit();
+    editorPage.typeTitle(article.title);
+    editorPage.typeDescription(article.description);
+    editorPage.typeBody(article.body);
+    editorPage.typeTags(article.tag);
+    editorPage.clickPublishBtn();
+    editorPage.assertArticleTitle(article.title);
+    editorPage.assertArticleBody(article.body);
   });
 
   it('should be edited using Edit button', () => {
+    const newbody = faker.lorem.words();
+    cy.get('@user').then((user) => {
```

Comment:
The use of `cy.get('@user')` is incorrect here. The alias `@user` is not set in the code. You should directly use the `user` variable obtained from the `generateUser` task.